benchmark/dns/lookup.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+const common = require('../common.js');
+const lookup = require('dns').lookup;
+
+const bench = common.createBenchmark(main, {
+  name: ['', '127.0.0.1', '::1'],
```

Comment:
I chose this list because they don't actually hit c-ares and so (in this particular case) it is a better measure of the forced async callback change. Also when you start passing in non-empty hostnames, the `n` value will need to be lowered quite a bit.